modules/islandora_binary_object_zip_importer/includes/importer.inc

```diff
@@ -208,6 +208,29 @@ class BinaryObjectZipBatchImportObject extends ZipBatchImportObject {
       $datastream['label'] = $this->getLabel($datastream['dsid'], $datastream);
     }
 
+    // See if the user uploaded DC, if not create it now.
+    if (!isset($to_return['DC'])) {
+      $obtained_dc = $this->getDC($name);
```

Comment:
Is... `$name` actually used? `getDC()` isn't implemented here, and it [doesn't appear to be used in the parent](https://github.com/Islandora/islandora_importer/blob/7.x/modules/zip_importer/includes/importer.inc#L461)...?